4-rc/alpine/Dockerfile

```diff
@@ -0,0 +1,73 @@
+# https://docs.ghost.org/faq/node-versions/
+# https://github.com/nodejs/LTS
+# https://github.com/TryGhost/Ghost/blob/3.3.0/package.json#L38
+FROM node:12-alpine3.12
+
+# grab su-exec for easy step-down from root
+RUN apk add --no-cache 'su-exec>=0.2'
+
+RUN apk add --no-cache \
+# add "bash" for "[["
+		bash
+
+ENV NODE_ENV production
+
+ENV GHOST_CLI_VERSION 1.15.3
+RUN set -eux; \
+	npm install -g "ghost-cli@$GHOST_CLI_VERSION"; \
+	npm cache clean --force
+
+ENV GHOST_INSTALL /var/lib/ghost
+ENV GHOST_CONTENT /var/lib/ghost/content
+
+ENV GHOST_VERSION 4.0.0-alpha.1
+
+RUN set -eux; \
+	mkdir -p "$GHOST_INSTALL"; \
+	chown node:node "$GHOST_INSTALL"; \
+	\
+	su-exec node ghost install "$GHOST_VERSION" --db sqlite3 --no-prompt --no-stack --no-setup --dir "$GHOST_INSTALL"; \
```

Comment:
this _should_ work now if you add `--channel next` to the arguments list

Reply:
I think I might've missed something because it's giving me `Message: Version 4.0.0 does not exist` :sweat_smile:

(from `ghost install 4.0.0-alpha.3 --channel next --db sqlite3 --no-prompt --no-stack --no-setup --dir /var/lib/ghost`)

Reply:
hmm that does appear to be a bug. I never tested specifying the exact alpha release to install 😅  will test and try to get a fix out

Reply:
I wonder if it makes sense just to wait until 4.0 is officially released before creating the docker image?

Reply:
:shrug: we're happy to go either way -- we like to publish pre-releases for the same reason the upstreams usually do (more users testing before GA, hopefully finding more bugs), but we're happy to defer :+1:

Reply:
@tianon so it looks like the reason this fails is because `semver.coerce()` doesn't like prerelease versions 🤔 https://github.com/npm/node-semver/issues/357

we cu